src/main/java/org/broadinstitute/consent/http/service/DatabaseDataAccessRequestAPI.java

```diff
@@ -562,12 +561,18 @@ public File createDataSetApprovedUsersDocument(Integer dataSetId) throws IOExcep
     @Override
     public DARModalDetailsDTO DARModalDetailsDTOBuilder(Document dar, DACUser dacUser, ElectionAPI electionApi) {
         DARModalDetailsDTO darModalDetailsDTO = new DARModalDetailsDTO();
-        List<Document> datasetDetails = populateDatasetDetailDocuments(dar.get(DarConstants.DATASET_DETAIL));
+        List<DataSet> datasets = populateDatasets(dar.get(DarConstants.DATASET_DETAIL));
+        Optional<DACUser> optionalUser = Optional.ofNullable(dacUser);
```

Comment:
Technically, this shouldn't be necessary, but through security testing, we've found that it's possible to inject invalid data (i.e. fake user ids) which throws off a lot of the API functions.